rust/tests/command_optimize.rs

```diff
@@ -311,15 +315,16 @@ async fn test_conflict_for_remove_actions() -> Result<(), Box<dyn Error>> {
     };
 
     let mut transaction = other_dt.create_transaction(None);
-    transaction.add_action(action::Action::remove(remove));
+    transaction.add_action(Action::remove(remove));
     transaction.commit(None, None).await?;
 
-    let maybe_metrics = plan.execute(&mut dt).await;
+    let maybe_metrics = plan.execute(dt.object_store()).await;
     assert!(maybe_metrics.is_err());
     assert_eq!(dt.version(), version + 1);
     Ok(())
 }
 
+#[ignore = "we do not yet re-try in operations commits."]
```

Comment:
I'm finally working up to doing the commits conflict resolution. so this will be re-enabled soonishly ..